twistlock/datadog_checks/twistlock/twistlock.py

```diff
@@ -308,7 +308,9 @@ def _report_service_check(self, data, prefix, tags=None, message=""):
 
     def _retrieve_json(self, path):
         url = self.config.url + path
-        response = self.http.get(url)
+        project = self.config.project
+        qparams = {'project': project} if project else None
+        response = self.http.get(url, params=qparams)
```

Comment:
Could you add a unit test for that ? 

Or piggy back on an existing test.

Reply:
I've added some form of test. I was hoping to get `response.request.path_url` but coulnd't since we mock the response. anyway, I;ve made some other changes. Please advise what tests are needed.